### DISCUSSION TITLE
Issue in setting up Sunbird ED : 'Credentials must be provided when creating a service client'

### DISCUSSION THREAD

**Discussion by GtAtEy** [Sunbird-Ed/Community#698] [Bugs]
Hi Team, 
When i am setting up the sunbird ED in my local system getting error 'Credentials must be provided when creating a service client.' while running the command - 'node server.js' in src/app directory. Please refer the screen shot.
![Screenshot from 2024-03-13 15-14-32](https://github.com/Sunbird-Ed/Community/assets/126556048/3b45d210-2d80-458e-9115-3d10e3e80cc3)

**Comment by gandham-santhosh** [comment]
@GtAtEy I understand that you are trying to setup Ed Portal on your local laptop/machine. can you please share the documentation link you are referring to ? cc: @rajeevsathish

**Reply by GtAtEy** [reply]
@paritshivani

**Comment by GtAtEy** [comment]
[NULP _ Sunbird ED setup doc.docx](https://github.com/Sunbird-Ed/Community/files/14646903/NULP._.Sunbird.ED.setup.doc.docx)
I have attached the document i am referring to

**Comment by vinukumar-vs** [comment]
@GtAtEy This is very old version of Sunbird-ED (Release- 4.10.0). We are currently in release-6.0.0

Sorry we say this, as a community will be supporting only previous release([release-5.2.0 stable](https://ed.sunbird.org/v/unsupported-versions/use/releases/release-notes/release-5.2.0))

By looking into the issue, we can see that some of the mandatory environment variables for Sunbird-Portal build are missing, especially those related to the Azure storage account. Please verify your enironemntVariables.js file used for the Sunbird-Portal build.

@rajeevsathish @mohitga @santhosh-tg

**Comment by HarishGangula** [comment]
As we there are no active updates, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
